### PR TITLE
feat(frontend): calculate max BTC amount on send

### PR DESCRIPTION
**Pull request by DenysKarmazynDFINITY**
# Motivation

Calculate max BTC amount on send, for now without fees.

### CODE REVIEW

File: src/frontend/src/btc/components/send/BtcSendAmount.svelte [hunk 1]
```diff
@@ -1,10 +1,32 @@
...
 	export let amountError: BtcAmountAssertionError | undefined;
+
+	const { sendToken, sendBalance } = getContext<SendContext>(SEND_CONTEXT_KEY);
+
+	$: calculateMax = (): number | undefined =>
+		isNullish($sendToken)
+			? undefined
+			: // TODO: Add fee to the calculation
+				getMaxTransactionAmount({
+					balance: $sendBalance ?? ZERO,
```
Comment (AntonioVentilii-DFINITY) [inline src/frontend/src/btc/components/send/BtcSendAmount.svelte:21]: i think you don't need a fallback here (it is included in the function)

I see that it is used with fallback in `SendForm`, but it is not needed there either

Comment (AntonioVentilii-DFINITY) [inline src/frontend/src/btc/components/send/BtcSendAmount.svelte:21]: i think you don't need a fallback here (it is included in the function)

I see that it is used with fallback in `SendForm`, but it is not needed there either

**Review by AntonioVentilii-DFINITY — APPROVED** [review]
LGTM!

Just check my comment before, but not a blocker

File: src/frontend/src/btc/components/send/BtcSendAmount.svelte (hunk 1 above)
Comment (DenysKarmazynDFINITY) [reply src/frontend/src/btc/components/send/BtcSendAmount.svelte]: True, I changed it to `sendBalance ?? undefined` to comply with types (`balance | null | undefined` -> `balance | undefined`).

**Pull request merged**